test/.nycrc-mocha

```diff
@@ -1,15 +1,16 @@
 {
   "all": true,
   "exclude": [
```

Comment:
Perhaps change this to `include: [ "src/" ]`?

Reply:
Yeah, I didn't want to miss files / future additions though.
If it works, what about `include src/ and /*.js (at root level) + exclude webpack.config*.js`?

Reply:
@xPaw, I've done that in 47eb0382. I'm not convinced of the result, you tell me which one you prefer and I'm fine either way. I guess it's easier to miss with a whitelist than an exclude-list, but no big deal.